### ISSUE TITLE
Enable running without using an aspell dictionary

### ISSUE THREAD

**Issue by cpeel** [DistributedProofreaders/pptext#10]
https://github.com/DistributedProofreaders/ppwb/issues/10 requested being able to run `pptext` without specifying a language and aspell isn't run / used. This will require `pptext` to be updated so `--aspell` is an optional parameter and doesn't default to `en`.

**Comment by tangledhelix** [comment]
**TL;DR: I propose we can add a new flag `-s` to disable aspell features; that way we can explicitly tell pptext to skip everything related to aspell (because using aspell requires a language be set).**

Notes on this, some taken from the ppwb ticket:

Skipping spellcheck is already possible. Spellcheck happens by default. If `-t a` is passed explicitly, all tests are run (including spellcheck). If `-t` is passed but its argument contains no `s` character, then spellcheck is skipped.

Not specifying a language is permitted: just don't pass an `-a` argument. But if you don't pass one, English is the implicit default. Currently it's not possible for pptext to run without a language specified (though you could specify one that doesn't exist, e.g. `-a xx`).

Even if spellcheck is skipped, aspell data is still used if the edit-distance check is run: `-t e`.

Even if spellcheck and edit-distance are both skipped, the punctuation scan uses aspell data for 3 tests:
- `in’` as a shortening of `ing` ending
- leading `’` as a dropped `h` as in `’ouse` for `house`
- trailing `s’` as the possessive form of a word ending in `s` (like `the dogs’` for "belongs to the dogs")

Those 3 tests make no sense except in English, however (at least so far as I know; it's possible one or more would apply to another supported language, but I'm not aware if so).

From what I can see, it seems like you can't do either 's' or 'e' modes without the aspell data. Obviously the spellcheck ('s') requires aspell. The edit-distance ('e') list of suspects comes from the aspell data; without a list of suspects, it won't do anything anyway.

As for the puncscan, it still does a lot of its work without aspell data. But it's not enough to check whether modes 's' or 'e' were used, because you're supposed to be able to run this check with mode 'q' solely too. And remember, if you don't specify a language, we default to English.

So I see two options:
1. If no language is specified (in PPWB), we could decline to call the 's', 'e', or 'q' tests. But technically the user is missing out on some of the work 'q' mode could do.
2. Or we could add a new mode with different behavior.

Conclusion: it might be best to add a new mode (by which I mean a CLI flag), let's call it `-s`. It would mean "turn off all aspell-dependent code". We can check for the mode to turn off entire sections of code (spellcheck, edit-distance), or specific tests within a section, as in the case of the 3 aspell tests done by the smartquote tool.

Adding a flag should mean we can do this without changing existing behaviors. Certain combinations of arguments would no longer make sense and should probably throw an exception (e.g. `-t e -s` or `-t s -s`)

**Comment by cpeel** [comment]
> **TL;DR: I propose we can add a new flag `-s` to disable aspell features; that way we can explicitly tell pptext to skip everything related to aspell (because using aspell requires a language be set).**

That makes a lot of sense to me. I'd rather have `pptext` manage this directly via a flag as you have proposed rather than encode that logic inside ppwb.
